gobblin-runtime/src/test/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiterTest.java

```diff
@@ -309,7 +309,7 @@ public void testReminderEventAcquireLeaseOnCompletedLease() throws IOException,
      Thread.sleep(MORE_THAN_EPSILON);
      // Now have a reminder event check-in on the completed lease
      LeaseAttemptStatus attemptStatus =
-         mysqlMultiActiveLeaseArbiter.tryAcquireLease(resumeDagAction, selectInfoResult.getEventTimeMillis(), true);
+         mysqlMultiActiveLeaseArbiter.tryAcquireLease(resumeDagAction, selectInfoResult.getEventTimeMillis(), true, false);
```

Comment:
Probably want to add one test where the the replacement boolean is set to true and validate that

Reply:
Forgot to commit changes, see test below now